core/services/loaders/ClassInterfaceCache.php

```diff
@@ -179,4 +179,18 @@ public function getFqnForAlias($alias = '', $for_class = '')
         }
         return $alias;
     }
+
+
+    // public function debug($for_class = '')
+    // {
+    //     if ($for_class !== '') {
+    //         if ( ! isset($this->aliases[ $for_class ])) {
+    //             \EEH_Debug_Tools::printr('NOT FOUND', "aliases[ $for_class ]", __FILE__, __LINE__);
+    //             return;
+    //         }
+    //         \EEH_Debug_Tools::printr($this->aliases[ $for_class ], "aliases[ $for_class ]", __FILE__, __LINE__);
+    //         return;
+    //     }
+    //     \EEH_Debug_Tools::printr($this->aliases, '$this->aliases', __FILE__, __LINE__);
+    // }
```

Comment:
Still needed ?

Reply:
it was handy, and i was trying to sneak it past you guys 😆 
but i can remove it

Reply:
Yeah, no need to remove then, just double checking. Maybe I'll need it myself 🙂